firmware/application/apps/acars_app.hpp

```diff
@@ -32,85 +32,69 @@
 #include "acars_packet.hpp"
 
 class ACARSLogger {
-public:
+ public:
 	Optional<File::Error> append(const std::string& filename) {
 		return log_file.append(filename);
 	}
-	
+
 	void log_raw_data(const acars::Packet& packet, const uint32_t frequency);
-	//void log_decoded(const acars::Packet& packet, const std::string text);
+	// void log_decoded(const acars::Packet& packet, const std::string text);
 
-private:
-	LogFile log_file { };
+ private:
+	LogFile log_file{};
 };
 
 namespace ui {
 
 class ACARSAppView : public View {
-public:
+ public:
 	ACARSAppView(NavigationView& nav);
 	~ACARSAppView();
 
 	void focus() override;
 
 	std::string title() const override { return "ACARS (WIP)"; };
 
-private:
-	bool logging { false };
-	uint32_t packet_counter { 0 };
+ private:
+	bool logging{false};
+	uint32_t packet_counter{0};
 
-	RFAmpField field_rf_amp {
-		{ 13 * 8, 0 * 16 }
-	};
-	LNAGainField field_lna {
-		{ 15 * 8, 0 * 16 }
-	};
-	VGAGainField field_vga {
-		{ 18 * 8, 0 * 16 }
-	};
-	RSSI rssi {
-		{ 21 * 8, 0, 6 * 8, 4 },
-	};
-	Channel channel {
-		{ 21 * 8, 5, 6 * 8, 4 },
+	RFAmpField field_rf_amp{{13 * 8, 0 * 16}};
+	LNAGainField field_lna{{15 * 8, 0 * 16}};
+	VGAGainField field_vga{{18 * 8, 0 * 16}};
+	RSSI rssi{
+			{21 * 8, 0, 6 * 8, 4},
```

Comment:
I wonder why it flakes out like this sometimes. The ones above it are fine.

Reply:
Yeah agree thus looks odd, but there would be some rule making it do this

Reply:
Ah, there's an unnecessary trailing comma there. Must be throwing off the parsing.